### ISSUE TITLE
Synchronize

### ISSUE THREAD

**Issue by Vohras2** [ObliqueNET/Terra#58]
Suljia and I tested with multiple synchronize pokemon, and it seems that none of them are setting the nature of wild pokemon during battle. I caught around 20 pokemon with a Kadabra as the lead, and not a single one had the nature. Suljia tested more with her ralts and had a similar result.

**Comment by fatyg5** [comment]
Not exactly a bug just a feature the team has disregarded for now. Might be added in the future.

**Comment by MattBlower** [comment]
This was discussed and the nature of the Pixelmon is generated on spawning, not on battle start

**Comment by MattBlower** [comment]
Been implemented so can be closed

**Comment by CptFuzzyFace** [comment]
Still having this issue on version 2.5.1 of Pixelmon Generations Modpack

**Comment by Vohras2** [comment]
> Still having this issue on version 2.5.1 of Pixelmon Generations Modpack

Pixelmon Generations has been using outdated code stolen from Pixelmon Reforged. I'm not surprised that they have issues with implementation. I'm afraid that we are unable to provide any support regarding a Generations modpack. If you would like to switch to the maintained Pixelmon Reforged, there is a discord available at https://discord.gg/Aadchbk. Mod information can be found at reforged.gg

**Comment by CptFuzzyFace** [comment]
Thanks for the response, was considering switching but wasn't sure how it would affect the save file.On Feb 21, 2019 3:05 PM, Vohras2 <notifications@github.com> wrote:
Still having this issue on version 2.5.1 of Pixelmon Generations Modpack

Pixelmon Generations has been using outdated code stolen from Pixelmon Reforged. I'm not surprised that they have issues with implementation. I'm afraid that we are unable to provide any support regarding a Generations modpack. If you would like to switch to the maintained Pixelmon Reforged, there is a discord available at https://discord.gg/Aadchbk. Mod information can be found at reforged.gg

**Comment by Hiroku** [comment]
How did you even find this bug tracker if you use Generations? I admire your reach but I have so many questions

**Comment by CptFuzzyFace** [comment]
It took a fair amount of stupid on my part. My friends and I wanted to play some pixelmon after not having played it since before the split. They immediately identified that reforged was the better mod but we didn't get around to playing at the time. About a week later I decided to impulse buy a server, and forgot which mod we had decided on.I couldn't really find indication of the difference between the mods from surface-level googling so I just went with generations, and then didn't realize that the host also supported reforged until recently, after we had all put 20+ hours into the world. After playing reforged for 5 minutes I can see that it's clearly the far better mod and I wish I could switch at this point. I appreciate your time even though this issue has really nothing to do with reforged.On Feb 22, 2019 4:17 AM, Hiroku <notifications@github.com> wrote:How did you even find this bug tracker if you use Generations? I admire your reach but I have so many questions

**Comment by Vohras2** [comment]
> It took a fair amount of stupid on my part. My friends and I wanted to play some pixelmon after not having played it since before the split. They immediately identified that reforged was the better mod but we didn't get around to playing at the time. About a week later I decided to impulse buy a server, and forgot which mod we had decided on.I couldn't really find indication of the difference between the mods from surface-level googling so I just went with generations, and then didn't realize that the host also supported reforged until recently, after we had all put 20+ hours into the world. After playing reforged for 5 minutes I can see that it's clearly the far better mod and I wish I could switch at this point. I appreciate your time even though this issue has really nothing to do with reforged.On Feb 22, 2019 4:17 AM, Hiroku <notifications@github.com> wrote:How did you even find this bug tracker if you use Generations? I admire your reach but I have so many questions —You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the threadOn Feb 22, 2019 4:17 AM, Hiroku <notifications@github.com> wrote:How did you even find this bug tracker if you use Generations? I admire your reach but I have so many questions —You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.

I'm like 99% sure that reforged supports swapping from Generations. I'd recommend going to the Reforged Discord and asking exactly how to.

**Comment by CptFuzzyFace** [comment]
I tried briefly but got a chunk error crash and world corruption after playing reforged for about a half hour. But thanks, I'll check out the discord. I appreciate the help!On Feb 22, 2019 7:56 AM, Vohras2 <notifications@github.com> wrote:

It took a fair amount of stupid on my part. My friends and I wanted to play some pixelmon after not having played it since before the split. They immediately identified that reforged was the better mod but we didn't get around to playing at the time. About a week later I decided to impulse buy a server, and forgot which mod we had decided on.I couldn't really find indication of the difference between the mods from surface-level googling so I just went with generations, and then didn't realize that the host also supported reforged until recently, after we had all put 20+ hours into the world. After playing reforged for 5 minutes I can see that it's clearly the far better mod and I wish I could switch at this point. I appreciate your time even though this issue has really nothing to do with reforged.On Feb 22, 2019 4:17 AM, Hiroku notifications@github.com wrote:How did you even find this bug tracker if you use Generations? I admire your reach but I have so many questions —You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the threadOn Feb 22, 2019 4:17 AM, Hiroku notifications@github.com wrote:How did you even find this bug tracker if you use Generations? I admire your reach but I have so many questions —You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.



I'm like 99% sure that reforged supports swapping from Generations. I'd recommend going to the Reforged Discord and asking exactly how to.
